migrations/900000000000012_add_gitleaks_sync.up.sql

```diff
@@ -0,0 +1,33 @@
+BEGIN;
+
+INSERT INTO mergestat.repo_sync_types (type, description, short_name) VALUES ('GITLEAKS_REPO_SCAN', 'Executes a gitleaks scan on a git repository', 'Gitleaks Repo Scan') ON CONFLICT DO NOTHING;
```

Comment:
If you don't define priority it will default to 0 and that will compete with things like "Sync Now" which we use for top priority.

Reply:
updated to default to `priority` `3`, like trivy and syft